src/stravalib/client.py

```diff
@@ -189,17 +195,50 @@ def exchange_code_for_token(
             The developer client secret
         code : str
             The temporary authorization code
+        return_athlete : bool (default = False)
+            Whether to return a SummaryAthlete object (or not)
+            This parameter is currently undocumented and could change
+            at any time.
 
         Returns
         -------
         dict
             Dictionary containing the access_token, refresh_token and
             expires_at (number of seconds since Epoch when the provided access
             token will expire)
+        tuple
+            Contains:
+             - the access token dict
+             - a `SummaryAthlete` object representing the authenticated user.
+
+        Notes
+        -----
+        Strava by default returns `SummaryAthlete` information during
+        this exchange. However this return is currently undocumented
+        and could change at any time.
         """
-        return self.protocol.exchange_code_for_token(
-            client_id=client_id, client_secret=client_secret, code=code
-        )
+        raw = self.protocol.exchange_code_for_token(
+            client_id=client_id,
+            client_secret=client_secret,
+            code=code,
+            return_athlete=return_athlete,
+        )
+        if return_athlete:
+            try:
+                athlete_info = SummaryAthlete.model_validate(raw[1])
```

Comment:
@jsamoocha we don't seem to use ValidationError in other places that I can find. But I wasn't sure how to handle the case where athlete disappears in the return. If we use get (which is great) it returns None. So when it tries to validate this will fail. 

in this case we could either only return AccessInfo object OR we could raise a warning that the endpoint return has changed. I chose to raise a warning AND also return only the AccessInfo object.

Do you think this is handled appropriately or should I raise an error or handle this differently if the athlete return is None?   many thanks!